solutions/src/main/java/ru/sberbank/school/task08/SerializableManager.java

```diff
@@ -17,17 +19,42 @@ public SerializableManager(@NonNull String filesDirectoryPath) {
 
     @Override
     public void initialize() {
-        throw new UnsupportedOperationException("Implement me!");
+        if (filesDirectory.equals("")) {
+            separator = "";
+        }
     }
 
     @Override
-    public void saveGame(String filename, Savable gameState) throws SaveGameException {
-        throw new UnsupportedOperationException("Implement me!");
+    public void saveGame(@NonNull String filename,
+                         @NonNull Savable gameState) throws SaveGameException {
+        try (FileOutputStream fos = new FileOutputStream(filesDirectory + separator + filename);
+             ObjectOutputStream oos = new ObjectOutputStream(fos)) {
```

Comment:
&#39;try resource&#39; has incorrect indentation level 13, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a706600479>*